src/engine/cuecontrol.cpp

```diff
@@ -17,6 +17,7 @@ static const double CUE_MODE_MIXXX = 0.0;
 static const double CUE_MODE_PIONEER = 1.0;
 static const double CUE_MODE_DENON = 2.0;
 static const double CUE_MODE_NUMARK = 3.0;
+static const double CUE_MODE_MIXXX_NO_BLINK = 4.0;
```

Comment:
Can you add this todo:
// TODO: Convert these doubles to a standard enum and convert elseif logic to switch statements.

That change can go in trunk, it's not a priority but it annoys me here.